pyproject.toml

```diff
@@ -12,7 +12,10 @@ dependencies = [
     "cloudpickle >=2.0.0",
     "etelemetry >=0.2.2",
     "filelock >=3.0.0",
+    "fileformats >=0.6",
```

Comment:
Do we need an upper bound in case of future API breakage (`<0.7` or `<1`)?

Reply:
Not sure, could be useful as fileformats is still evolving but then I don't have much experience with upper bounds and how they play out in practice.

Reply:
You can use `fileformats >=0.6,<0.7` if you don't guarantee API breakages between `0.x`versions. `fileformats >=0.6,<1` may be useful if you allow yourself API breakages right before committing to version 1.

Reply:
Ok, I have done >=0.8,<0.9 just in case for now.